test-utils/junit5-extension/src/main/java/org/operaton/bpm/engine/test/junit5/ParameterizedTestExtension.java

```diff
@@ -77,6 +80,12 @@
  */
 public class ParameterizedTestExtension implements TestTemplateInvocationContextProvider {
 
+	@Retention(RetentionPolicy.RUNTIME)
```

Comment:
Nice extension of `ParameterizedTestExtension`

When you are on it: Please make the javadocs proper HTML with `<ol><li>...` for lists , `<pre>` for code blocks.
And add something about the parameters.

Reply:
Will do.